CHANGELOG.md

```diff
@@ -84,6 +84,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Dependencies
 
 ### Changed
+- Enable `./gradlew build` on MacOS by disabling bcw tests ([#7303](https://github.com/opensearch-project/OpenSearch/pull/7303))
```

Comment:
@jed326 the changelog was cleared for 2.8.0, you just need your change here (other changes should not be there):

```
### Changed
- Add min, max, average and thread info to resource stats in tasks API ([#7673](https://github.com/opensearch-project/OpenSearch/pull/7673))
```

Reply:
@reta ahhh I see. Fixed this in the latest revision. Thanks!

Reply:
> @reta ahhh I see. Fixed this in the latest revision. Thanks!

Sorry, another conflict :(